src/pint/models/jump.py

```diff
@@ -2,86 +2,84 @@
 import logging
 
 import astropy.units as u
-import numpy
+import numpy as np
 
 from pint.models.parameter import maskParameter
 from pint.models.timing_model import DelayComponent, MissingParameter, PhaseComponent
 
 log = logging.getLogger(__name__)
 
-
-class DelayJump(DelayComponent):
```

Comment:
Tempo actually has delay jumps for the observatory cable delays. I was thinking to switch the backend jump to delay jump. The jump value affects the binary delay calculation.